### ISSUE TITLE
Mobile app: Slack attachment button color looks wrong color for dark themes

### ISSUE THREAD

**Issue by mickmister** [mattermost/mattermost#25810]
There was an issue in the webapp where he slack attachment buttons had the wrong background for dark themes. It was reported that this is occurring on the mobile app as well. Here is the original report for the mobile app issue:

---

I'm seeing that default style button is hard to see on mobile with Indigo and Onyx theme. (server v9.3 / mobile v2.11.0)

| Mobile | Desktop |
|:-:|:-:|
| <img width="500" src="https://github.com/mattermost/mattermost/assets/1453749/d2e72ef5-3ed8-43e0-a8ab-6074bb8c679d"> | <img width="500" alt="image" src="https://github.com/mattermost/mattermost/assets/1453749/79753fef-aeec-4e54-866e-73eaa9c61e97"> |

_Originally posted by @kaakaa in https://github.com/mattermost/mattermost/issues/23968#issuecomment-1864469111_

**Comment by MustkimKhatik** [comment]
Hey can I grab this issue :) ?

**Comment by mickmister** [comment]
Hi @MustkimKhatik! Absolutely, thank you!

**Comment by sumedhakoranga** [comment]
If this issue is still pending, can I get assigned for this?

**Comment by mickmister** [comment]
Hi @sumedhakoranga, thanks for your interest in contributing! Yes it seems the ticket is available. I'll assign you now :+1:

@MustkimKhatik Please let me know if you were still wanting to work on the ticket. Thank you!

**Comment by mickmister** [comment]
@sumedhakoranga Here's some instructions on how to create a post like the one shown in the description https://gist.github.com/mickmister/6d1bbcfe386c833ede9fede37525f6f5?permalink_comment_id=4705211#gistcomment-4705211

**Comment by mickmister** [comment]
Hi @sumedhakoranga, I hope you're doing well

Just checking in, are you still wanting to work on this ticket? Thank you for the interest in contributing 🙂

**Comment by sumedhakoranga** [comment]
Hi, yes, I am working on this. I will send PR  by the end of this week.

**Comment by mickmister** [comment]
Thanks @sumedhakoranga! Let me know if you have any questions. There is a similar PR that addressed similar issues in the webapp https://github.com/mattermost/mattermost/pull/24628. There may be some good inspiration there for solving this issue

**Comment by sumedhakoranga** [comment]
Hi, I am getting this error:  Cannot invoke "java.io.File.getAbsolutePath()" because the return value of "com.google.gct.studio.appengine.sdk.AppEngineSdk.getToolsApiJarFile()" is null,

I've checked to ensure the App Engine SDK is correctly configured in my project settings.
I'm hoping someone can help me understand why getToolsApiJarFile() is returning null and how I might resolve this issue to proceed with my project.

Thank you in advance for your assistance!

**Comment by mickmister** [comment]
@larkox Do you have any thoughts on the above issue?

**Comment by larkox** [comment]
@sumedhakoranga Some sanity check:
- When are you seeing this issue? While running the app or while installing?
- Are you running master?
- Have you run `npm i` before `npm run android`?
- Just to make sure, run `npm run clean` and then `npm i`

**Comment by sumedhakoranga** [comment]
Hi, @larkox 
 - I started by cloning the repository to ensure I'm working with the most up-to-date version of the code.
 - Switched my Node.js environment to version 18.10.0 to match the project's requirements.
 - I ran npm install
 - I executed npm run android
 
 Issue Encountered: Upon running npm run android, I received a warning regarding an invalid configuration in the react-native-math-view package, mentioning that "dependency.assets" is not allowed. Despite this warning, the JS server starts, but I am concerned this may affect the app's functionality or indicate a deeper configuration issue.
 Could you please advise on how to address this warning or if there are any specific steps I should follow to resolve this issue?

**Comment by larkox** [comment]
@sumedhakoranga Yes, that error can be ignored. It has to do with the library we use to render latex formulas, but should not create any issue.

**Comment by nikkivaddepelli** [comment]
I would like to work on this issue. Can you please assign this to me?

**Comment by ladudu** [comment]
Hi @mickmister 
Could you please assign this to me?
The issue may be that the ActionButton component is not applying the intended background color (customColor), and instead it's hardcoded  to #ffffff.
